### ISSUE TITLE
Uncaught TypeError: Cannot read property 'slice' of undefined

### ISSUE THREAD

**Issue by jgirardet** [pghilardi/atom-python-test#16]
[Enter steps to reproduce:]

Right click on a def test in a class test
Run test under cursor

**Atom**: 1.20.0 x64
**Electron**: 1.6.9
**OS**: Linux Mint
**Thrown From**: [atom-python-test](https://github.com/pghilardi/atom-python-test) package 0.7.0

### Stack Trace

Uncaught TypeError: Cannot read property 'slice' of undefined

```
At /home/XXXX/.atom/packages/atom-python-test/lib/atom-python-test.coffee:99

TypeError: Cannot read property 'slice' of undefined
    at Object.runTestUnderCursor (/packages/atom-python-test/lib/atom-python-test.coffee:99:34)
    at /packages/atom-python-test/lib/atom-python-test.coffee:35:106)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/usr/share/atom/resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/usr/share/atom/resources/app/src/atom-environment.js:1344:34)
    at EventEmitter.outerCallback (/usr/share/atom/resources/app/src/application-delegate.js:348:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -4:36.5.0 intentions:highlight (input.hidden-input)
     -4:36.1.0 atom-python-test:run-test-under-cursor (input.hidden-input)
  2x -4:19.7.0 core:backspace (input.hidden-input)
     -4:16.8.0 intentions:highlight (input.hidden-input)
     -4:16.6.0 core:save (input.hidden-input)
     -4:13.8.0 atom-python-test:run-all-tests (input.hidden-input)
     -3:42.1.0 tree-view:copy (span.name.icon.icon-file-text)
     -1:05.6.0 core:backspace (input.hidden-input)
     -1:01.0 intentions:highlight (input.hidden-input)
     -1:00.7.0 core:save (input.hidden-input)
     -0:55.3.0 atom-python-test:run-all-tests (input.hidden-input)
  4x -0:38.9.0 core:backspace (input.hidden-input)
     -0:36.9.0 intentions:highlight (input.hidden-input)
     -0:36.6.0 core:save (input.hidden-input)
     -0:30 atom-python-test:run-all-tests (input.hidden-input)
     -0:02.7.0 atom-python-test:run-test-under-cursor (span.syntax--entity.syntax--name.syntax--function.syntax--python)
```

### Non-Core Packages

```
atom-python-test 0.7.0 
atom-python-virtualenv 0.14.0 
busy-signal 1.4.3 
intentions 1.1.5 
linter 2.2.0 
linter-flake8 2.2.1 
linter-ui-default 1.6.8 
python-autopep8 0.1.3 
```

**Comment by hasaziz** [comment]
any idea when this will get fixed?
thks

**Comment by pghilardi** [comment]
Fixed in version 0.8.0. Please, download the latest version @hasaziz .

Thanks for the contribution @jgirardet !
